Jenkinsfile

```diff
@@ -0,0 +1,23 @@
+#!groovy
+
+pipeline {
+  agent none
+  stages {
+    stage('Maven Install') {
+      agent {
+        docker {
+          image 'maven:3.5.0'
```

Comment:
why not to use the latest one?
https://hub.docker.com/layers/library/maven/3.9.1/images/sha256-dcfe7934d3780beda6e1f0f641bc7db3fa9b8818899981f6eddae052c78394c7?context=explore